lib/libutee/tee_api.c

```diff
@@ -607,14 +607,14 @@ const void *TEE_GetInstanceData(void)
 	return tee_api_instance_data;
 }
 
-void *TEE_MemMove(void *dest, const void *src, size_t size)
+void TEE_MemMove(void *dest, const void *src, size_t size)
 {
-	return memmove(dest, src, size);
+	(void)memmove(dest, src, size);
```

Comment:
Please drop the cast, same below.

Reply:
No problem. I removed the cast in https://github.com/OP-TEE/optee_os/pull/7140/commits/312fd9960ca51a8172adad2fdcef218c97ca5ae7.

Reply:
Hug?! I did not know that fixup commits also needed to be signed. Regardless, fixed again in https://github.com/OP-TEE/optee_os/pull/7140/commits/311d28d1320f385ffefcc15736f2e669ecfab0ac.